pkg/config/config.go

```diff
@@ -114,6 +114,7 @@ type CongestionControlConfig struct {
 	UseSendSideBWE     bool                       `yaml:"send_side_bandwidth_estimation,omitempty"`
 	ProbeMode          CongestionControlProbeMode `yaml:"padding_mode,omitempty"`
 	MinChannelCapacity int64                      `yaml:"min_channel_capacity,omitempty"`
+	UseTWCC            bool                       `yaml:"use_twcc"`
```

Comment:
Using a new flag for TWCC. `UseSendSideBWE` is still there for using with Pion's GCC. Leaving both in for now. Will clean up later. Also, it is enabled below. That is just in my test branch. Will turn that off before merging.